### DISCUSSION TITLE
[Esame 30 Giugno 2014] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#246] [Esami - Bertini @ Matematica]
![Screenshot 2023-11-14 164622](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/f5baf595-8f18-4372-a1d8-b1f75adb7e5b)
![Screenshot 2023-11-14 164650](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/b11fa7b7-9ad1-4304-ad43-499127120e23)

**Comment by cornflexxx** [comment]
$i)$ Distribuzione di $U$ : 

$$\mathbb{P}(U>u)=\mathbb{P}(X> u , Y> u)=2\mathbb{P}(X>u)=2(1-\mathbb{P}(X\le u))=(1-p)^{2z+2}$$

$$\mathbb{P}(U=u)=(1-p)^{2z+2}-(1-p)^{2z}$$

$ii)$ Distribuzione di $V$:

$$\mathbb{P}(V=v)=\mathbb{P}(X-Y=v)=\mathbb{P}(X=v-y, Y=y)=\mathbb{P}(X=v-y)\cdot \mathbb{P}(Y=y)=p^2(1-p)^{v+2y-2}$$
